frontend/src/toolbar/bar/Toolbar.scss

```diff
@@ -102,7 +102,9 @@
 .Toolbar {
     --toolbar-button-x: 0;
     --toolbar-button-y: 0;
-    --toolbar-translate: translate(var(--toolbar-button-x), var(--toolbar-button-y));
+    --toolbar-width-minimized: 40px;
+    --toolbar-width-expanded: 292px;
```

Comment:
Will this need to be updated every time we add a new element to the toolbar? Thinking about the new experiments icon. Think that's behind a feature flag which might confuse this further. cc @Phanatic